.github/workflows/ci.yml

```diff
@@ -19,11 +19,35 @@ jobs:
               uses: actions/setup-python@v2
               with:
                   python-version: ${{ matrix.python-version }}
-            - run: |
+            - name: Install dependencies
+              run: |
                 sudo apt update
-                sudo apt install libdbus-1-dev libgirepository1.0-dev gir1.2-gtk-3.0 gir1.2-notify-0.7 gir1.2-gudev-1.0 graphviz imagemagick x11-apps xserver-xephyr xterm xvfb libpulse-dev lm-sensors
+                sudo apt install libdbus-1-dev libgirepository1.0-dev gir1.2-gtk-3.0 gir1.2-notify-0.7 gir1.2-gudev-1.0 \
+                  graphviz imagemagick x11-apps xserver-xephyr xterm xvfb libpulse-dev lm-sensors libwlroots-dev git ninja-build
                 pip install tox tox-gh-actions
-              name: install dependencies
+                # we want wlroots' dependencies only, so remove the packages (these are outdated in ubuntu's repos)
+                sudo dpkg -r --force-depends wlroots libwayland-dev
```

Comment:
Not related to the mypy changes, but do you also need to remove `libwlroots-dev` and all the `libwayland-*` packages?

Reply:
I believe so. (lib)wayland is built manually because it is also packaged by ubuntu with an outdated version that is incompatible with wlroots 0.13